boards/xg23b-pk6068a/include/board.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2022 SSV Software Systems GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_xg23b-pk6068a
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the xG23B-PK6068A
+ *
+ * @author      Juergen Fitschen <me@jue.yt>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Initialize pm_layered with no pre-defined blockers
+ *
+ * All peripheral drivers keep track of required pm modes.
+ */
+#define PM_BLOCKER_INITIAL { 0, 0, 0 }
+
+/**
+ * @name    ztimer configuration
+ * @{
+ */
+#define CONFIG_ZTIMER_LPTIMER_DEV            TIMER_DEV(1)
+#define CONFIG_ZTIMER_LPTIMER_FREQ           LFXO_FREQ
+#define CONFIG_ZTIMER_LPTIMER_WIDTH          24
+#define CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE  -1
+/** @} */
+
+/**
+ * @brief   Power mode required for GPIO IRQs
+ *
+ * If all GPIO IRQs are expected on port A or B, EM3 is the lowest allowed
+ * power mode. No power level has to be blocked in this case.
+ * If GPIO IRQs shall be fired on port C or D, we must stay in EM1.
+ * Uncomment the line down below in this case.
+ */
+//#define GPIO_INT_PM_BLOCKER PM_MODE_EM2
+
+/**
+ * @name    Board VCOM interface
+ *
+ * Define the GPIO pin to enable the VCOM interface
+ */
+#define VCOM_UART           UART_DEV(0)
+#define VCOM_EN_PIN         GPIO_PIN(PB, 0)
+
+/**
+ * @name    Push button pin definitions
+ * @{
+ */
+#define PB0_PIN             GPIO_PIN(PB, 1)
+#define PB1_PIN             GPIO_PIN(PB, 3)
+/** @} */
+
+/**
+ * @name    LED pin definitions
+ * @{
+ */
+#define LED0_PIN            GPIO_PIN(PB, 2)
+#define LED1_PIN            GPIO_PIN(PD, 3)
+/** @} */
+
+/**
+ * @name    Macros for controlling the on-board LEDs
+ * @{
+ */
+#define LED0_ON             gpio_set(LED0_PIN)
+#define LED0_OFF            gpio_clear(LED0_PIN)
+#define LED0_TOGGLE         gpio_toggle(LED0_PIN)
+#define LED1_ON             gpio_set(LED1_PIN)
+#define LED1_OFF            gpio_clear(LED1_PIN)
+#define LED1_TOGGLE         gpio_toggle(LED1_PIN)
+/** @} */
+
+/**
+ * @name    Display configuration
+ *
+ * Connection to the on-board Sharp Memory LCD (LS013B7DH03).
+ * @{
+ */
+#define DISP_SPI            SPI_DEV(0)
+#define DISP_CS_PIN         GPIO_PIN(PC, 8)
```

Comment:
Should be called as in the Board users guide 
```suggestion
#define DISP_SCS_PIN        GPIO_PIN(PC, 8)
```

Reply:
https://github.com/RIOT-OS/RIOT/pull/18780/commits/1581e2915d5ead8043eb5ef566e29bcef84d29ea